src/Resources/views/orderRefunds.html.twig

```diff
@@ -129,10 +129,11 @@
                         </tbody>
                     </table>
 
-                    <label for="payment-methods" style="font-weight: bold;">{{ 'sylius.ui.payment_method'|trans }}</label>
+                    {% set original_payment_method = order.payments.first().method %}
+                    <label for="payment-methods" style="font-weight: bold;">{{ 'sylius.ui.payment_method'|trans }} ({{ 'sylius.ui.original_payment_method'|trans }}: {{ original_payment_method }})</label>
```

Comment:
Could you please provide some screenshot how does it look right now?

Reply:
![orig_payment_method](https://user-images.githubusercontent.com/1793860/48696340-f5466380-ebe1-11e8-88f8-4c5e522f39e1.png)